locales/ro/ro.json

```diff
@@ -364,9 +364,9 @@
     "Mauritius": "Mauritius",
     "May": "Mai",
     "Mayotte": "Mayotte",
+    "Meeting": "Întalnire",
```

Comment:
No need to add your own words. This package does an automatic comparison of all keys.

If you need to create your own set of translations, you can use the [`template`](https://github.com/Laravel-Lang/translations-template).